plugin.xml

```diff
@@ -81,11 +81,6 @@
 
     <!-- android -->
     <platform name="android">
-        <hook type="after_plugin_add" src="scripts/android-install.js" />
-        <hook type="after_plugin_install" src="scripts/android-install.js" />
-        <hook type="before_plugin_rm" src="scripts/android-install.js" />
-        <hook type="before_plugin_uninstall" src="scripts/android-install.js" />
```

Comment:
I missed this in the migration to Capacitor - this script was only getting called in certain lifecycle hooks in Cordova.

This script is in charge of taking EntryActivity.java, WXEntryActvity.java, and WXPayEntryActivity.java and moving them into (for us) `android/src/main/java/com/smartly/hybrid` _and also_ replacing the string literal `__PACKAGE_NAME__` with `com.smartly.hybrid`.